<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>244</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Open Encrypted Volume</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="PushButtonOpen">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReadOnly">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount In &amp;Read Only Mode</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonMountPointPath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>select mount point path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonVolumePath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open volume path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassPhraseField">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPassPhraseFromFile">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open key file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="OpenVolumePath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MountPointPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassphrase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMoutPointPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlugin">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbKeyOption">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKeyType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Key</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Key+KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plugin</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cbVeraCryptVolume">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;VeraCrypt Volume</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbShareMountPoint">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Share Mount Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="veraCryptWarning">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>OpenVolumePath</tabstop>
  <tabstop>PushButtonVolumePath</tabstop>
  <tabstop>MountPointPath</tabstop>
  <tabstop>PushButtonMountPointPath</tabstop>
  <tabstop>checkBoxReadOnly</tabstop>
  <tabstop>cbKeyType</tabstop>
  <tabstop>pushButtonPlugin</tabstop>
  <tabstop>PassPhraseField</tabstop>
  <tabstop>pbKeyOption</tabstop>
  <tabstop>pushButtonPassPhraseFromFile</tabstop>
  <tabstop>PushButtonOpen</tabstop>
  <tabstop>PushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
